dev/prod/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "module": "esnext",
     "target": "es6",
     "allowJs": true,
+    "sourceMap": true,
```

Comment:
I believe we need to have source maps for packages translated to js... this one will not help. Or it helps?

Reply:
It helps to debug actually from VSCode, not in browser unfortunately, so some research is required.